src/models/Script/index.ts

```diff
@@ -3,3 +3,8 @@ export interface Script {
   args: string
   hashType: string
 }
+
+export interface OutPoint {
```

Comment:
It seems that it should be placed in `models/Transaction`, it is the same type as `CellDep['outPoint']`.

Reply:
Any update on this comment

Reply:
done